homeassistant/components/homematicip_cloud/__init__.py

```diff
@@ -63,6 +63,8 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
                 )
             )
 
+    await async_setup_services(hass)
```

Comment:
Note: we now recommend to keep the services registered, and you should consider dropping `async_unload_services` in `homematicip_cloud`